database/migrations/2021_01_01_100000_create_leads_table.php

```diff
@@ -13,16 +13,16 @@ public function up()
             $table->id();
             $table->string('lead_number')->index()->unique(); // Generated by system. This is identifier used to communicate about the quote request & for additional form pages.
             $table->string('email');
-            $table->foreignIdFor(LeadBusiness::class);
+            $table->foreignIdFor(LeadBusiness::class); // moving to other table to reverse the relationship
             $table->string('first_name')->nullable();
             $table->string('last_name')->nullable();
             $table->string('position')->nullable();
             $table->string('phone_number')->nullable(); // Will be replaced later with relationship to Phone model in tipoff/addresses
             
-            // Move these 3 fields to LeadBusiness table
-            $table->boolean('current_plan_under_cancellation')->nullable();
-            $table->date('current_plan_expires_at')->nullable();
-            $table->text('past_comp_claims')->nullable();
+                // Move these 3 fields to LeadBusiness table
+                $table->boolean('current_plan_under_cancellation')->nullable();
+                $table->date('current_plan_expires_at')->nullable();
+                $table->text('past_comp_claims')->nullable();
```

Comment:
@gurgentil Can you move these 3 fields to the lead_businesses table?

Reply:
Yes!